docs/docs/modeling.md

```diff
@@ -133,6 +133,7 @@ The TenantsWork contains AssignedCategory. The TenantsWork class has AssignedCat
 Constraints:
 
 - The TenantsWork cannot exist without any AssignedCategory.
```

Comment:
So it is. I think this constraint is not for the `AssignedCategory`, but for the `TenantsWork`.